templates/email/default/problem-confirm-not-sending.html

```diff
@@ -22,7 +22,7 @@ <h1 style="[% h1_style %]">Please confirm your&nbsp;report</h1>
   <p style="[% p_style %]">If you no longer wish to publish this report, please take no further action.</p>
   [% end_padded_box %]
 </th>
-[% WRAPPER '_email_sidebar.html' object = report %]
+[% WRAPPER '_email_sidebar.html' object = report, url = token_url %]
```

Comment:
TBH the comma does make it clearer, doesn't it (unless it was `object=report url=token_url` I guess). So feel free to leave it, I don't mind.